packages/map/src/CdcMap.js

```diff
@@ -82,6 +82,15 @@ const hashObj = row => {
   }
 }
 
+const indexOfIgnoreType = (arr, item) => {
```

Comment:
This makes sense to me. You might be able to use `Array.prototype.findIndex` here:

`const indexOfIgnoreType = (arr, item) => arr.findIndex(element => element === item);`

A few additional things to keep in mind:
- type checking the array
- use strict equality checks
- variable names (getIndexOf, findIndexOf, etc.), without the ticket in front of me indexOfIgnoreType sounds confusing.

Let me know if you want to make any updates here or if we should just pass it through.